<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Пользователи</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="2" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Доступ</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="lvl_cb"/>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLineEdit" name="login_e"/>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLineEdit" name="pass_e"/>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTableWidget" name="user_table"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="add_b">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="del_b">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="back_b">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
